Formula/gutenberg.rb

```diff
@@ -0,0 +1,39 @@
+class Gutenberg < Formula
+  desc "Opinionated static site generator with everything built-in"
+  homepage "https://www.getgutenberg.io/"
+  url "https://github.com/Keats/gutenberg/archive/v0.2.1.tar.gz"
+  sha256 "407097164b1d6807e7dca94d68fba7538b0a85f8cf08448c4a5ca3357cfd00ca"
+  head "https://github.com/Keats/gutenberg.git"
+
+  depends_on "cmake" => :build
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "build", "--release"
+
+    bin.install "target/release/gutenberg"
+    bash_completion.install "completions/gutenberg.bash-completion"
+    zsh_completion.install "completions/_gutenberg"
+    fish_completion.install "completions/gutenberg.fish"
+  end
+
+  test do
+    pipe_output("#{bin}/gutenberg init mysite", "\nn\nn\n")
```

Comment:
exit code is not 0?

Reply:
I always forget it’s not the default. Fixed.